project.clj

```diff
@@ -8,7 +8,8 @@
                  ^:source-dep [cheshire "5.8.0"]
                  ^:source-dep [alembic "0.3.2"]
                  ^:source-dep [org.clojure/tools.analyzer.jvm "0.7.1"]
-                 ^:source-dep [org.clojure/tools.namespace  "0.3.0-alpha3"]
+                 ^:source-dep [org.clojure/tools.namespace "0.3.0-alpha3"]
+                 ^:source-dep [org.clojure/java.classpath "0.2.3"]
```

Comment:
This dep should not be needed. The project should only use the classpath functionality by provided by `orchard`.

Reply:
I think @SevereOverfl0w just forgot the delete this when he added orchard.

Reply:
I'll try removing both dependencies in that case, thanks Bozhidar

Reply:
Looking at the source for cider-nrepl: https://github.com/clojure-emacs/cider-nrepl/blob/2e7be20d3ed9afb0ef952529bbb1b7eace6712da/project.clj#L20

I believe this is the current best way to work around the original problem (from the issue), orchard isn't inlining those dependencies so to avoid conflicts the higher level library (cider-nrepl, refactor-nrepl) needs to inline them (by inline I mean source-dep via mranderson).

I could have misunderstood the problem but when I've removed those dependencies as mentioned before I get the following:

```
Caused by: java.io.FileNotFoundException: Could not locate clojure/java/classpath__init.class or clojure/java/classpath.clj on classpath.
        at clojure.lang.RT.load(RT.java:449)
        at clojure.lang.RT.load(RT.java:412)
        at clojure.core$load$fn__5448.invoke(core.clj:5866)
        at clojure.core$load.doInvoke(core.clj:5865)
        at clojure.lang.RestFn.invoke(RestFn.java:408)
        at clojure.core$load_one.invoke(core.clj:5671)
        at clojure.core$load_lib$fn__5397.invoke(core.clj:5711)
        at clojure.core$load_lib.doInvoke(core.clj:5710)
        at clojure.lang.RestFn.applyTo(RestFn.java:142)
        at clojure.core$apply.invoke(core.clj:632)
        at clojure.core$load_libs.doInvoke(core.clj:5749)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.core$apply.invoke(core.clj:632)
        at clojure.core$require.doInvoke(core.clj:5832)
        at clojure.lang.RestFn.invoke(RestFn.java:482)
        at mranderson048.orchard.v0v3v0.toolsnamespace.v0v3v0_alpha4.clojure.tools.namespace.find$eval1925$loading__5340__auto____1926.invoke(find.clj:9)
        at mranderson048.orchard.v0v3v0.toolsnamespace.v0v3v0_alpha4.clojure.tools.namespace.find$eval1925.invoke(find.clj:9)
        at clojure.lang.Compiler.eval(Compiler.java:6782)
        at clojure.lang.Compiler.eval(Compiler.java:6771)
        at clojure.lang.Compiler.load(Compiler.java:7227)
        ... 77 more
```